release-notes/unreleased/IGAPP-411-fix-menu-bar-on-small-screens.yml

```diff
@@ -0,0 +1,7 @@
+issue_key: IGAPP-363
+show_in_stores: true
+platforms:
+  - android
+  - ios
+en: Hide redundant navigation items.
+de: Redundante Navigationselemente werden nun nicht mehr angezeigt.
```

Comment:
Nothing against copy and paste but I guess you should change this :D

Reply:
How does this happen?? Sorry. I fixed it.